spec/spec_helper.rb

```diff
@@ -1,3 +1,4 @@
 require './book'
+require './music_album'
```

Comment:
- Please try to change `require` to `require_relative` to avoid inconsistency issues.